responsive-article/gulcin/styles.css

```diff
@@ -0,0 +1,396 @@
+* {
+    box-sizing: border-box;
```

Comment:
CSS looks good, general advice:
- try to stick to BEM for class names
- use variables for things that you have to repeat: colors, boder-radius, border-width. Set them up at the very top so that you always know what are all the colors that are used in the project. It safes a lot of time!
- Try to always put the most generic selectors at the very top of the CSS file and then move to more generic selectors (I think I saw h1 selector pretty low in the file) - saves time :)

Reply:
I didn't know much about BEM and CSS variables while writing this code. Im trying to stick to BEM for my projects now. Thank you!